src/views/report/ScriptReportDetail/index.tsx

```diff
@@ -253,6 +253,12 @@ function ExecutionDetail({ state }: IObserveProps) {
 
         const executionListItems: IDescriptionListItem[] = [
             ...(scriptExecutionData && asyncExecutionRequest.status === AsyncStatus.Success) ? [
+                {
+                    label: translator('script_reports.detail.side.execution.requestor.label'),
+                    value: scriptExecutionData.username.length
+                        ? <Translate msg={scriptExecutionData.username} />
```

Comment:
the actual value of scriptExecutionData.username should not be translated

Reply:
Ya Robbe, I have tested and we are getting the actual value of Requester

Reply:
Okay, but isn't that because if Translate does not find a translation it returns the same value by default?
you're trying to translate the username and that will never be possible